cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -655,10 +722,11 @@ void ImageFlip::disparityCallback(const stereo_msgs::DisparityImage::ConstPtr& d
 				dst--;
 			}
 		}
-		rot_mat.at<double>(0,0) = -1.;
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.